Lib/lib2to3/fixer_util.py

```diff
@@ -150,6 +150,39 @@ def ImportAndCall(node, results, names):
     new.prefix = node.prefix
     return new
 
+def BlankLineOrPass(node):
+    """Returns either a blank line or a pass statement depending on
+    the node's parent's siblings to maintain syntactic correctness
+    within a suite after conversion"""
+    skip = {token.NEWLINE, token.INDENT, token.DEDENT}
+    def has_significant_sibling(node, is_forward):
+        sibling = None
```

Comment:
Is this definition necessary? You are changing the value of sibling without actually needing this.

Reply:
`has_significant_sibling` is only a helper function to check parent's siblings in both directions and does not perform any modification to the parent's node. I wrapped it in a function so that it will be easier to read the logic to decide whether to return a blank line or `pass` statement:

```
parent = node.parent
if parent and parent.parent and parent.parent.type == syms.suite:
    if (not has_significant_sibling(parent, False)
        and not has_significant_sibling(parent, True)):
        return Name("pass")
return BlankLine()
```

Reply:
I meant `sibling = None`

Reply:
Okay yes that makes sense. I've updated the code to remove that line. Thanks for the review!